File: tests/fixtures/phase1/safety.ts
// Résumé 😀 comment prose.
const value = "strings are protected";

/**
 * Documentation prose is eligible.
 * @example
 * const untouched = "example code";
 */
export function example(): string {
  // spellagent-disable-next-line
  // This comment is disabled.
  return value;
}

File: tests/fixtures/phase1/safety.py
"""Module résumé 😀 prose.

>>> print("protected doctest")
"""

value = """This non-docstring string is protected."""

def example():
    """Function documentation prose."""
    # Eligible comment prose.
    return value

def escaped():
    """An escaped docstring is skipped: \N{SNOWMAN}."""
